pycytominer/cyto_utils/cells.py

```diff
@@ -41,7 +42,11 @@ class SingleCells(object):
         Columns indicating how to merge image and compartment data.
     image_cols : list of str, default ["TableNumber", "ImageNumber", "Metadata_Site"]
         Columns to select from the image table.
-    feature: str or list of str, default "infer"
+    image_features : list of str, optional
+        List of category of features from the image table to add to the profiles.
+    add_image_features: bool, default False
```

Comment:
because `add_image_features` isn't a user argument (it is set by side effect) and you specify it here, I think you'll also want to add `image_feature_categories` and describe it here too

Reply:
Good point. I changed `image_features` to `image_feature_categories` and refactored the code a bit - https://github.com/cytomining/pycytominer/pull/168/commits/805b290bff90236696e8626e5494fad6dfd177e8